File: samples/server/petstore/jaxrs/src/test/java/io/swagger/client/auth/Authentication.java
package io.swagger.client.auth;

import io.swagger.client.Pair;

import java.util.Map;
import java.util.List;

@javax.annotation.Generated(value = "class io.swagger.codegen.languages.JavaClientCodegen", date = "2015-08-23T11:08:48.917-07:00")
public interface Authentication {
  /** Apply authentication settings to header and query params. */
  void applyToParams(List<Pair> queryParams, Map<String, String> headerParams);
}

File: samples/client/petstore/kotlin-multiplatform/src/common/main/org/openapitools/client/auth/Authentication.kt
package org.openapitools.client.auth

import org.openapitools.client.infrastructure.Queries

interface Authentication {
    /**
     * Apply authentication settings to header and query params.
     *
     * @param queries Query parameters.
     * @param headers Header parameters.
     */
    fun apply(queries: Queries, headers: MutableMap<String, String?>)
}
